src/manual/tasks/artifacts.md

```diff
@@ -0,0 +1,50 @@
+---
+filename: tasks/artifacts.md
+title: Artifacts
+order: 35
+---
+
+By far the most commonly used data from a run are its artifacts. These are
+HTTP entities containing output data from the task execution.
+
+Unlike most API methods which return a JSON body, requesting an artifact from
+the Queue service returns the artifact itself, possibly via one or more HTTP
+redirects. This means that -- at least for public artifacts which require no
+authentication -- any sufficiently robust HTTP client can download an artifact
+directly.
+
+**NOTE**: Not all clients are "sufficiently robust"! The artifact interface
```

Comment:
I actually don't know a way to do this with a common tool.  And we don't need signed URLs in this context (talking about public artifacts), which saves a little bit of complexity in the explanation.  Do you have an example of a robust approach?